actionview/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   Alias `ActionView::Helpers::TagHelper#class_names` to `dom_token_list` and
+    ensure the resulting Token list items are unique.
```

Comment:
Since this is replacing the existing change log entry, how about something along these lines?

```suggestion
*  Add `dom_token_list` helper to create token lists with conditional values.
```